src/app/workflow/versions/versions.component.html

```diff
@@ -15,170 +15,164 @@
-            <label placement="top" matTooltip="A version is valid if it has at least one valid descriptor file">
-              Valid
-            </label>
-          </th>
+    <ng-container matColumnDef="hidden">
+      <th
+        scope="col"
+        mat-header-cell
+        *matHeaderCellDef
+        mat-sort-header
+        matTooltip="A hidden version is only visible here and not publicly."
+      >
+        Hidden
+      </th>
+      <td mat-cell *matCellDef="let version">
+        <mat-icon *ngIf="version.hidden">check</mat-icon>
+      </td>
+    </ng-container>
 
-          <th *ngIf="!publicPage">
-            <fa-icon class="pull-right" [icon]="getIconClass('hidden')" (click)="clickSortColumn('hidden')"></fa-icon>
-            <label placement="top" matTooltip="A hidden version is only visible here and not publicly.">
-              Hidden
-            </label>
-          </th>
+    <ng-container matColumnDef="verified">
+      <th
+        scope="col"
+        mat-header-cell
+        *matHeaderCellDef
+        matTooltip="A version is verified if it has been verified to work by a person/group."
+      >
+        Verified Platforms
+      </th>
+      <td mat-cell *matCellDef="let version">
+        <a href="{{ verifiedLink }}" *ngIf="version.verified" matTooltip="{{ getVerifiedSource(version.name) }}">
+          <mat-icon>check</mat-icon>
+        </a>
+        {{ version?.sourceFiles | verifiedPlatforms }}
+      </td>
+    </ng-container>
 
-          <th>
-            <fa-icon class="pull-right" [icon]="getIconClass('verified')" (click)="clickSortColumn('verified')"></fa-icon>
-            <label placement="top" matTooltip="A version is verified if it has been verified to work by a person/group">
-              Verified Platforms
-            </label>
-          </th>
-          <th>
-            <label
-              placement="top"
-              matTooltip="The metadata and descriptors for a version that has been snapshotted will not change over time."
-            >
-              Snapshot
-            </label>
-          </th>
-          <th>
-            <label placement="top" matTooltip="View more information about the given version">
-              Actions
-            </label>
-          </th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr *ngIf="versions?.length === 0">
-          <td colspan="8">
-            <div class="text-center">No Records Found</div>
-          </td>
-        </tr>
-        <tr
-          (click)="setVersion(version)"
-          *ngFor="let version of versions | orderBy: convertSorting(workflow?.mode)"
-          [ngClass]="{ info: _selectedVersion != null && version != null && _selectedVersion.id == version.id }"
+    <ng-container matColumnDef="snapshot">
+      <th
+        scope="col"
+        mat-header-cell
+        *matHeaderCellDef
+        matTooltip="The metadata and descriptors for a version that has been snapshotted will not change over time."
```

Comment:
It was also like this, but the metadata, or at least some of it, can change after it's been snapshotted.